cves/CVE-2019-14232.yml

```diff
@@ -122,10 +134,12 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This bug was reported directly to the Django by two developers. It appears that
+    Guido Vranken hunts vulnerabilities as a hobby.
```

Comment:
Consider adding a link to where Guido Vraken is being credited

Reply:
Agreed, give credit where credit's due